main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAttributionSchema.java

```diff
@@ -0,0 +1,39 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+@GtfsTable("attributions.txt")
+public interface GtfsAttributionSchema extends GtfsEntity {
+    @FieldType(FieldTypeEnum.ID)
+    @PrimaryKey
+    String attributionId();
+
+    @FieldType(FieldTypeEnum.ID)
+    @ForeignKey(table = "agency.txt", field = "agency_id")
+    String agencyId();
+
+    @FieldType(FieldTypeEnum.ID)
+    @ForeignKey(table = "routes.txt", field = "route_id")
+    String routeId();
+
+    @ForeignKey(table = "trips.txt", field = "trip_id")
```

Comment:
Should also be `@FieldType(FieldTypeEnum.ID)`?

Reply:
Done.